common-compose.yml

```diff
@@ -0,0 +1,70 @@
+wordpress:
```

Comment:
@roXet: https://github.com/autopilotpattern/couchbase may be the canonical example for this. I explained in https://github.com/autopilotpattern/mysql/issues/12:

> Ideally we'd like to eliminate use of the `--link` to Consul and use DNS discovery instead. To make this easier, a [`setup.sh` script](https://github.com/autopilotpattern/couchbase/blob/master/setup.sh) like the one in https://github.com/autopilotpattern/couchbase would be a convenient addition that could [write out the `_env` file](https://github.com/autopilotpattern/couchbase/blob/master/setup.sh#L105) with the DNS name for the Consul service.
>
> The [`demo.sh` script`](https://github.com/autopilotpattern/couchbase/blob/master/demo.sh) is not strictly required, but it automatically opens up the web web browser to the correct locations, making for a convenient demo.